core/model-api/src/main/java/org/eclipse/rdf4j/model/ValueFactory.java

```diff
@@ -1,10 +1,10 @@
 /*******************************************************************************
- * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
+ * Copyright (c) ${year} Eclipse RDF4J contributors.
```

Comment:
Your IDE forgot to substitute the number here. Also: I don't think we should replace the license header on this just because we moved the file.